signer/src/testing/wsts.rs

```diff
@@ -468,6 +468,7 @@ impl SignerSet {
     }
 
     /// Dump the current signer set as a dummy rotate-keys transaction to the given storage
+    /// `None` for `signatures_required` defaults to the number of signers
```

Comment:
nit: this comment addition is a bit unclear: there is no `signatures_required` argument, and it seems the function always create a rotate key event from `shares.signature_share_threshold` that cannot be none. What is it referring to?